.github/workflows/main.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
             container-image: "stakater-reloader"
           - dir: "./trust-packages"
             container-image: "trust-packages"
+          - dir: "./pod-delete-rate-limiter"
+            container-image: "pod-delete-rate-limiter"
```

Comment:
Will the test is run in CI?

Reply:
added a option to run `make test` before building  a image outside of `docker build`.